.github/workflows/deploy.yml

```diff
@@ -0,0 +1,65 @@
+name: deploy
+
+"on":
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
We don't need to trigger on pull requests, so remove this one. We're deploying only on "pushes" to "main".